frontend/src/utils/numbers.ts

```diff
@@ -8,3 +8,11 @@ export function maybeParseFloat(string: string | null) {
   }
   return float
 }
+
+/**
+ * Convenience function for rounding without using toFixed (which is for formatting).
```

Comment:
What the difference in usage? Is formatting not enough in this case?

Reply:
`toFixed` gives you a string, so rounding and stringifying all the populations just to parse them back into floats again is expensive and defeats the point a little bit. Though that said, this is just for the CSV so maybe them being stringy is fine - I just lifted the function out of the other approach where I was trying to do the rounding much lower down (which required them to be numbers).

Reply:
Convinced